docs/html/user_guide.rst

```diff
@@ -791,3 +791,25 @@ of ability. Some examples that you could consider include:
 
 * ``distlib`` - Packaging and distribution utilities (including functions for
   interacting with PyPI).
+
+
+.. _`Disable Keyring`:
+
+Disable Keyring
+***************
+
+Keyring in certain cases can prevent the installing of certain packages and modules
```

Comment:
Perhaps we can indicate how a user could recognize if they are impacted by this issue? The language here is a bit vague, perhaps there are some example warnings (maybe they are `ResourceWarning`s or `Exception`s that would indicate if `keyring` is causing this problem).

Reply:
By default you don't get any output unfortunately. If you increase the verbosity you are going to read something like `Getting credentials from keyring for https://pypi.org/simple` before it get stuck